components/agenda/src/Agenda.svelte

```diff
@@ -568,14 +568,19 @@
     setTimeout(() => {
       scrolling = false;
     }, 500);
+
+    // Only allow zooming in when meta or ctrl key is pressed while scrolling
+    if (!event.metaKey && !event.ctrlKey) {
+      return;
+    }
```

Comment:
I would prefer if this was handled as a property; as in `zoom_behavior: "default" | "ctrl"` or something similar.

Reply:
(I like zoom as the default mechanism. I think we're QAing this as a full-screen component, but I expect that that will be the minority of use-cases. I think embedding this alongside other items on a page will be much more common)